src/test/scala/chiselTests/AutoCloneTypeSpec.scala

```diff
@@ -0,0 +1,332 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chiselTests
+
+import chisel3._
+import chisel3.testers.TestUtils
+
+class BundleWithIntArg(val i: Int) extends Bundle {
```

Comment:
Can these (or most, or some) of these be inlined / placed closer to the test in which they are used, so tests are a bit more readable? For example, this only seems to be used in one place.

Reply:
These are the original tests for reflective autoclonetype. The tests I added start at line 256. They need to be left alone for the time being since they're still being used by the "noPluginTests."

Reply:
I'm going to undo the file rename to make the diff more clear.